src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -36,6 +43,15 @@ const BookingModalForm = props => {
       : 'Day'
 }`;
 
+    const labelTextValue = `${
+      isEventCancelled
+        ? 'Cancelled'
+        : rejectionMessages
+          ? 'Submit'
+          : `Update to ${bookingDuration === 0.5 ? 'Half' : bookingDuration}
+      ${bookingDuration > 1 ? 'Days' : 'Day'}`
```

Comment:
This still isn't very clear haha, nesting ternaries wherever they may appear isn't very readable.

```
const getLabelTextValue = () => {
  if (isEventCancelled) { return 'Cancelled'; }
  if (rejectionMessages) { return 'Submit'; }

  const durationText = bookingDuration === 0.5 ? 'Half' : bookingDuration;
  const dayText = bookingDuration > 1 ? 'Days' : 'Day';

  return `Update to ${durationText} ${dayText}`;
}
```